### PR TITLE
update core: scoped dms

### PR DESCRIPTION
**Pull request by tarcisiozf**
enables scoped DMS for electron app

plus: externalize config for remote DMS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -124,7 +124,7 @@
     "bfx-hf-ext-plugin-bitfinex": "git+https://github.com/bitfinexcom/bfx-hf-ext-plugin-bitfinex.git#v1.0.11",
     "bfx-hf-models": "git+https://github.com/bitfinexcom/bfx-hf-models.git#v2.4.1",
     "bfx-hf-models-adapter-lowdb": "git+https://github.com/bitfinexcom/bfx-hf-models-adapter-lowdb.git#v1.0.5",
-    "bfx-hf-server": "git+https://github.com/bitfinexcom/bfx-hf-server.git#v4.18.0",
+    "bfx-hf-server": "git+https://github.com/tarcisiozf/bfx-hf-server.git#external-dms",
```
Comment (tarcisiozf) [inline package.json:127]: wait for merge

**Review by bhoomij — APPROVED** [review]
looks, good. 
@tarcisiozf could you help me understand what is bfxHostedWsUrl ?

**Comment by tarcisiozf** [comment]
> looks, good. @tarcisiozf could you help me understand what is bfxHostedWsUrl ?

This is one argument exposed by hf-server so we can externally override the address for the remote dms, like using local env. So I need to update on the electron for dev purposes too.

File: package-lock.json [hunk 2]
```diff
@@ -2214,6 +2214,7 @@
     "bfx-hf-server": {
       "version": "git+https://github.com/bitfinexcom/bfx-hf-server.git#64f20fd27623ea428516b9e51fd0474f5fdde407",
       "from": "git+https://github.com/bitfinexcom/bfx-hf-server.git#v4.19.0",
+
```
Comment (vigan-abd) [inline package-lock.json:2217]:
```suggestion
```

**Review by vigan-abd — APPROVED** [review]

**Pull request merged**